gdal/ogr/ogrsf_frmts/s57/s57featuredefns.cpp

```diff
@@ -265,6 +265,15 @@ S57GenerateVectorPrimitiveFeatureDefn( int nRCNM,
 
     oField.Set( "RUIN", OFTInteger, 2, 0 );
     poFDefn->AddFieldDefn( &oField );
+	
+/* -------------------------------------------------------------------- */
+/*      Geometric primitive attributes                                  */
+/* -------------------------------------------------------------------- */	
+	oField.Set( "POSACC", OFTReal, 10, 2 );
```

Comment:
Indentation issues. Please use 4 spaces for each indentation level, and no space.